bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/HDPowerViewBindingConstants.java

```diff
@@ -67,8 +67,8 @@ public class HDPowerViewBindingConstants {
     // Shade properties
     public static final String PROPERTY_SHADE_TYPE = "type";
     public static final String PROPERTY_SHADE_CAPABILITIES = "capabilities";
-    public static final String PROPERTY_SECONDARY_RAIL_DETECTED = "secondaryRailDetected";
-    public static final String PROPERTY_TILT_ANYWHERE_DETECTED = "tiltAnywhereDetected";
+    public static final String PROPERTY_SECONDARY_RAIL_DETECTED = "secondaryRailDetected"; // hidden
+    public static final String PROPERTY_TILT_ANYWHERE_DETECTED = "tiltAnywhereDetected"; // hidden
```

Comment:
This is a bit obscure/confusing. I would suggest to remove the constants from this class and use a different terminology than  "properties". I see an additional advantage in doing that: We can get rid of the **soft** vs. **hard** properties concept.

How about `detectedCapabilities` or something like that? We could then rename method `updateHardProperties` to `updateDetectedCapabilities` and `updateSoftProperties` to simply `updateProperties`.

Reply:
Ok.

Reply:
Done.

Reply:
OK, I reject this comment as it's a bit more specific than that - and covers only type/capabilities, not all properties.